src/wallet/rpcdump.cpp

```diff
@@ -756,8 +756,8 @@ UniValue dumpwallet(const JSONRPCRequest& request)
 
     EnsureWalletIsUnlocked(pwallet);
 
-    fs::path filepath = request.params[0].get_str();
-    filepath = fs::absolute(filepath);
+    fs::path filepath = fs::absolute(request.params[0].get_str());
+    EnsureFileWritable(filepath.string());
```

Comment:
can remove the `.string()` here now

Reply:
Oops, I just removed a similar one from blockchain.cpp. Sorry. Will force push again. Thanks!